components/components/list_view.tsx

```diff
@@ -0,0 +1,62 @@
+import { GenerateList } from "./generate_list";
+import { useEffect, useState } from "react";
+import { NEXT_PUBLIC_AUTH_TOKEN } from "../../constants";
+
+let stat = "uptime data unavailable!";
+
+const ListView = function () {
+  const [dataList, setDataList] = useState([]);
+  const [loaded, setLoaded] = useState(false);
+  let idList=[];
+  let pages: any;
+
+  const getPageID = async () => {
+    const URL = "https://api.statuspage.io/v1/pages/"
+    const response = await fetch(URL, {
+      headers : {
+        "Content-Type": "application/json",
+        Authorization: `OAuth ${NEXT_PUBLIC_AUTH_TOKEN ?? ""}`
+      }
+    })
+    let xjson = await response.json();
+    idList = xjson.map(data=>data.id);
+    return idList; 
+  }
+
+  const getComponents = async () => {
+    pages = await  getPageID();
+    let urls = pages.map(page => {
+      return `https://api.statuspage.io/v1/pages/${page}/components`;
+    })
```

Comment:
since we only have single page right now please hard code the page id and avoid fetching all the pages.